### PR TITLE
Feature/main-page/styles

### PR DESCRIPTION
**Pull request by AnnaIstochnikova**

### CODE REVIEW

File: src/sass/main.scss [hunk 1]
```diff
@@ -5,3 +5,10 @@
 
 // Style dla pliku z przykładem HTML fragmentu
 @import './example';
```
Comment (krystianklimczak) [inline src/sass/main.scss:7]: Try to keep code clear deleting unnecessary line of code

File: src/sass/main.scss [hunk 2]
```diff
@@ -5,3 +5,10 @@
...
+
+@import './partials/_button';
+@import './partials/_commons';
+@import './partials/_container';
+@import './partials/_typography';
```
Comment (krystianklimczak) [inline src/sass/main.scss:12]: same here

**Review by krystianklimczak — APPROVED** [review]
For me is all fine. Ony one think I would change to make this button of trailer make more centered but it's not import right now. Great Job 💪

**Review by annienf17 — APPROVED** [review]

File: src/sass/main.scss (hunk 1 above)
Comment (krystianklimczak) [reply src/sass/main.scss]: done

**Pull request merged**